setup.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""Setup script."""
+
+# Copyright (c) 2022 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at 
+# <http://www.apache.org/licenses/LICENSE-2.0>
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from setuptools import find_packages, setup
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+with open('requirements.txt') as req_file:
+    requirements = req_file.read().splitlines()
+
+with open('requirements-test.txt') as req_file:
+    test_requirements = req_file.read().splitlines()[2:]
+
+with open('requirements-dev.txt') as req_file:
+    dev_requirements = test_requirements + req_file.read().splitlines()[2:]
+
+tox_requirements = [
+    'tox',
+    'tox-pyenv'
+]
+
+
+setup(
+    name='imagenameparser',
+    version='1.0.0',
```

Comment:
We should get the version from the version file otherwise we invite drift or setup.py must be part of the bumpversion setup.